openweathermap-ios.xcodeproj/xcuserdata/felipe.xcuserdatad/xcschemes/xcschememanagement.plist

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Consider ignoring all files under `xcuserdata`. 

It may help you: https://www.gitignore.io/api/swift,cocoapods